library/src/main/res/layout/month.xml

```diff
@@ -21,6 +21,7 @@
       android:layout_height="wrap_content"
       >
     <com.squareup.timessquare.CalendarRowView
+        android:id="@+id/week_row_view"
```

Comment:
let's call this something else: `day_names_header_row`?

Reply:
definitely a better name! Will do!